pages/api/resources.js

```diff
@@ -30,35 +30,24 @@ export default async (req, res) => {
     const searchTerm = req.query.searchTerm;
     const resourceTypes = req.query.resourceTypes;
 
-    let filter = {};
+    let params = buildPaginationParams(req);
 
     if (searchTerm && resourceTypes) {
       const resources = resourceTypes.split(",");
-      filter = {
-        filter: `name.contains("${searchTerm}")&&(${resources
-          .map((type) => `"type"=="${type}"`)
-          .join("||")})`,
-      };
+      params.filter = `name.contains("${searchTerm}")&&(${resources
+        .map((type) => `"type"=="${type}"`)
```

Comment:
Shouldn't make a difference either way, but the quotes aren't technically required
```suggestion
        .map((type) => `type=="${type}"`)
```